OmgvaPOS/src/BusinessManagement/Services/BusinessService.cs

```diff
@@ -11,13 +12,15 @@ public class BusinessService(IBusinessRepository businessRepository) : IBusiness
         public List<BusinessDTO> GetBusinesses() {
             return _businessRepository.GetBusinesses().Select(b => b.ToBusinessDTO()).ToList();
         }
-        public BusinessDTO GetBusiness(long id) {
+        public BusinessDTO? GetBusiness(long id) {
             return _businessRepository.GetBusiness(id).ToBusinessDTO();
```

Comment:
Does toBusinessDTO properly handle possible null?

Reply:
Will make sure this and other Mappers do, good point

Reply:
Resolved by making sure we don't try to convert null to DTO